app/src/main/java/org/oppia/android/app/player/state/ConfettiConfig.kt

```diff
@@ -0,0 +1,92 @@
+package org.oppia.android.app.player.state
+
+import android.content.Context
+import androidx.core.content.ContextCompat.getColor
+import nl.dionsegijn.konfetti.KonfettiView
+import nl.dionsegijn.konfetti.models.Shape
+import nl.dionsegijn.konfetti.models.Size
+import org.oppia.android.R
+
+/** A sealed class defining the standard configurations for various confetti animations */
+sealed class ConfettiConfig() {
+  internal abstract val context: Context
```

Comment:
Why internal? It seems like many of these should be protected, or passed into the constructor.

Reply:
Ah this was originally called for by the assembler so it couldn't be protected, will adjust. It is passed into the constructor of the subclasses, unless you mean the constructor for the ConfettiConfig?

Reply:
Done